packages/commonwealth/client/scripts/state/api/comments/createReaction.ts

```diff
@@ -0,0 +1,100 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import axios from 'axios';
+import { modelReactionCountFromServer } from 'controllers/server/comments';
+import app from 'state';
+import { ApiEndpoints } from 'state/api/config';
+import useFetchCommentReactionsQuery from './fetchReaction';
+
+interface CreateReactionProps {
+  address: string;
+  reactionType?: 'like';
+  chainId: string;
+  commentId: number;
+}
+
+const createReaction = async ({
+  address,
+  reactionType = 'like',
+  chainId,
+  commentId
+}: CreateReactionProps) => {
+  // TODO: use canvas id
```

Comment:
as I can see this comment is copied from old solution. Maybe it is good time to figure out if this is still necessary or we can remove this comment?

Reply:
Not sure what the comment means. cc @raykyri safe to remove?

Reply:
Yeah, fine to remove.

Reply:
done